src/client/admin.go

```diff
@@ -21,8 +21,12 @@ func (c APIClient) InspectCluster() (*admin.ClusterInfo, error) {
 }
 
 // Extract all cluster state, call f with each operation.
-func (c APIClient) Extract(objects bool, f func(op *admin.Op) error) error {
-	extractClient, err := c.AdminAPIClient.Extract(c.Ctx(), &admin.ExtractRequest{NoObjects: !objects})
+func (c APIClient) Extract(objects, enterprise, auth bool, f func(op *admin.Op) error) error {
+	extractClient, err := c.AdminAPIClient.Extract(c.Ctx(), &admin.ExtractRequest{
```

Comment:
Added an option to not extract the enterprise token or auth token, as a safety valve. With these disabled extract/restore is the same as previous releases.